frontend-ssr/components/feedback-review/CommentBlock.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Link from 'next/link';
+import { Comment } from './comment.model';
+import styles from './comment-block.module.scss';
+
+export default function CommentBlock(props: Comment) {
+  return (
+    <div className={styles.comment}>
+      <header>
+        <h4>
+          <Link href="/bob" prefetch={false}>
+            <a>
+              <strong>{props.user}</strong>
+            </a>
+          </Link>
+          <span> a lasat urmatorul comentariu</span>
+        </h4>
+        <button>&#128077;</button>
```

Comment:
Maybe use icons, like FontAwesome? Also, don't forget to add `type="button"`. Otherwise, inside forms it will default to `submit`.

Reply:
Done. I've added the FontAwesome packages so that we can use their icons in the project